src/main/java/salesforce/ui/pages/contract/ContractClassicPage.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Jalasoft.
+ */
+
+package salesforce.ui.pages.contract;
+
+import salesforce.ui.pages.newopportunity.AbstractNewOpportunityPage;
+
+/**
+ * Defines ContractClassicPage.
+ *
+ * @author Alan Escalera.
+ * @version 1.0 28 March 2020.
+ */
+public class ContractClassicPage extends AbstractContractPage {
+    @Override
+    protected void waitUntilPageObjectIsLoaded() {
+
+    }
+
+    @Override
+    public AbstractNewOpportunityPage clickOnNewOpportunity() {
+        //To Do
```

Comment:
delete this comment if the return is null

Reply:
I can't because it will throw an error

Reply:
read again the comment

Reply:
the observation was delete the comment